### PR TITLE
crosschain swaps: handle transaction confirmation status

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes TEAM2-496
Figma link (if any):

## What changed (plus any additional context for devs)

Originally i thought the issue on the ticket was produced because we weren't parsing txs correctly for crosschain swaps but the issue on the ticket was produced because zerion txs were overwriting txs on state, so crosschain swaps happening from mainnet as source network 

## Screen recordings / screenshots

https://www.loom.com/share/93fae4714d5a453580a27aa11bb4f189

In the video you'll see a mainnet <> OP swap that is not bridged yet, so the tx is on "Swapping" state while the bridge is confirming
- i didn't add android since i'd have to pay for another swap on mainnet and this is only a data change

## What to test

- send a mainnet <> L2 swap and you shouldn't see a "Sent Token" tx confirmed below the pending "Swapping" tx described on the ticket

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [ ] Did you test both iOS and Android?
- [ ] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Comment by estebanmino** [comment]
don't review it just yet, i found something hard testing it

**Comment by estebanmino** [comment]
ok we're good now https://www.loom.com/share/4441ffb322894cc68d8b4b512522471c including android

**Review by derHowie — APPROVED** [review]
fix looks good, might need another rebase, though. I'm seeing common commits between this and the base branch.

PoW: https://recordit.co/fnwKK2GqkN

File: src/parsers/transactions.ts [hunk 1]
```diff
@@ -270,6 +289,19 @@ const overrideTradeRefund = (txn: ZerionTransaction): ZerionTransaction => {
   };
 };
 
+const overrideSwap = (tx: ZerionTransaction): ZerionTransaction => {
```
Comment (skylarbarrera) [inline src/parsers/transactions.ts:292]: 👨‍🍳

**Review by skylarbarrera — APPROVED** [review]
working great

**Pull request merged**